From: Dan Williams <dan.j.williams@intel.com>
Date: Thu, 29 Jun 2017 09:02:10 -0700
Subject: acpi, nfit: quiet invalid block-aperture-region warnings
Patch-mainline: v4.13-rc1
Git-commit: d5d51fece79eafcdbc69d1836c28a3c9c7c8e862
References: bsc#1091781

This state is already visible by userspace since the BLK region will not
be enabled, and it is otherwise benign as it usually indicates that the
DIMM is not configured.

Signed-off-by: Dan Williams <dan.j.williams@intel.com>
Acked-by: Johannes Thumshirn <jthumshirn@suse.de>
---
 drivers/nvdimm/region_devs.c | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/drivers/nvdimm/region_devs.c b/drivers/nvdimm/region_devs.c
index 282b8991ea83..ab141f8b5140 100644
--- a/drivers/nvdimm/region_devs.c
+++ b/drivers/nvdimm/region_devs.c
@@ -833,7 +833,7 @@ int nd_blk_region_init(struct nd_region *nd_region)
 		return 0;
 
 	if (nd_region->ndr_mappings < 1) {
-		dev_err(dev, "invalid BLK region\n");
+		dev_dbg(dev, "invalid BLK region\n");
 		return -ENXIO;
 	}
 
-- 
2.12.3

